src/formats/MMTF.cpp

```diff
@@ -64,10 +67,12 @@ void MMTFFormat::read_step(const size_t step, Frame& frame) {
 void MMTFFormat::read(Frame& frame) {
     frame.resize(0);
 
-    UnitCell cell (file_->unitCell[0], file_->unitCell[1], file_->unitCell[2],
-                   file_->unitCell[3], file_->unitCell[4], file_->unitCell[5]);
+    if (file_->unitCell.size() >= 6) {
```

Comment:
Do you know in which cases could unitCell.size() be more than 6?

Reply:
It's not possible, I'll change it.

Reply:
Can it be less than 6 though ? I think 1 and 3 could make sense, but I don't know what the MTTF format says on this

Reply:
It says it must be 6 or zero, it is based on the CRYST1 line from PDB files I think.  The C++ code actually checks for this [here](https://github.com/rcsb/mmtf-cpp/blob/18fa57ceee1332da4f69f2998731013db9febd50/include/mmtf/structure_data.hpp#L460) so we know if the vector is size 0 (not given) or 6.

Reply:
ok, thanks!